CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# How to Contribute
+
+We'd love to accept your patches and contributions to this project.
```

Comment:
I'm holding off a bit on this one. Checking with Hans for capacity on his team. We might have to phrase this more conservatively in the interim.

Reply:
Sure thing, I just used the provided template for this, so suggestions for changes are welcome.

Reply:
This is fine by me, but obviously wait to see what Hans thinks.

Reply:
I don't have an answer for this yet, so let's just say we're not taking contributions for this project currently.

Reply:
Done.